jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/projects/jakarta-sample/src/main/java/io/openliberty/sample/jakarta/jsonb/JsonbTransientDiagnostic.java

```diff
@@ -32,18 +30,49 @@ public class JsonbTransientDiagnostic {
     @JsonbTransient
     private String favoriteLanguage;    // Diagnostic: JsonbTransient is mutually exclusive with other JsonB annotations
     
+    // No diagnostic as field is not annotated with other Jsonb annotations,
+    // even though the accessors are annotated with @JsonbTransient
+    private String favoriteDatabase;
+    
+    // Diagnostic will appear as field accessors have @JsonbTransient,
+    // but field itself has annotation other than transient
+    @JsonbProperty("fav_editor")
+    private String favoriteEditor;
+    
     @JsonbProperty("person-id")
-    @JsonbNillable
     private int getId() { 
         // A diagnostic is expected on getId because as a getter, it is annotated with other 
         // Jsonb annotations while its corresponding field id is annotated with JsonbTransient
         return id;
     }
     
-    @JsonbNillable
+    @JsonbNillable 
     private void setId(int id) {
         // A diagnostic is expected on setId because as a setter, it is annotated with other 
         // Jsonb annotations while its corresponding field id is annotated with JsonbTransient
         this.id = id;
     }
-}
+    
+    @JsonbTransient
+    private String getFavoriteDatabse() {
```

Comment:
This is a negative test case but I think it is going to be affected by the difference in spelling between the field name and the accessor name, "favoriteDatabase" vs "favoriteDatabse" (getFavoriteDatabse).

Reply:
Done. Thank you for catching that! I keep making typos :(